Apps/Sandcastle/gallery/CZML Circles and Ellipses.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">  <!-- Use Chrome Frame in IE -->
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="CZML Circles and Ellipses">
+    <meta name="cesium-sandcastle-labels" content="CZML">
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.9/require.js"></script>
+    <script type="text/javascript">
+        require.config({
+            baseUrl : '../../../Source',
+            waitSeconds : 60
+        });
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar"></div>
+
+<script id="cesium_sandcastle_script">
+    function startup(Cesium) {
+        "use strict";
+        //Sandcastle_Begin
+        var czml = [
+            {
+                "id" : "document",
+                "name" : "CZML Geometries: Circles and Ellipses",
+                "version" : "1.0"
+            },
+            {
+                "id" : "Green circle at height",
+                name: "Green circle at height",
```

Comment:
Ok, I will go through and fix those.

Once I do, how do I update the pull request?

Thanks,
Adam

On Tue, Sep 29, 2015 at 11:09 AM, Patrick Cozzi <notifications@github.com>
wrote:

> In Apps/Sandcastle/gallery/CZML Circles and Ellipses.html
> <https://github.com/AnalyticalGraphicsInc/cesium/pull/3050#discussion_r40684026>
> :
>
> > +<div id="loadingOverlay"><h1>Loading...</h1></div>
> > +<div id="toolbar"></div>
> > +
> > +<script id="cesium_sandcastle_script">
> > +    function startup(Cesium) {
> > +        "use strict";
> > +        //Sandcastle_Begin
> > +        var czml = [
> > +            {
> > +                "id" : "document",
> > +                "name" : "CZML Geometries: Circles and Ellipses",
> > +                "version" : "1.0"
> > +            },
> > +            {
> > +                "id" : "Green circle at height",
> > +                name: "Green circle at height",
>
> Throughout all examples, please use strict JSON so name should be "name".
> In general, all CZML property names should be strings in double quotes so
> it is easy to store them as JSON.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/AnalyticalGraphicsInc/cesium/pull/3050/files#r40684026>
> .
>